### DISCUSSION TITLE
Episode 176: `prior`, `deltas` & Dinner with Phineas

### DISCUSSION THREAD

**Discussion by codereport** [codereport/adsp2#68] [Episode]
Episode Link: https://adspthepodcast.com/2024/04/05/Episode-176.html

Feel free to write comments or ask questions 🙂

**Comment by CharString** [comment]
After the live coding sessions and the excellent Tacit series, this is way way up there in the "what are they talking about" axis... It took a while before I understood what that chez lounge was, that Bryce was talking about. I only knew the thing as a chaise longue, in European French pronunciation. https://www.youtube.com/watch?v=nHJuy7Ws3m8

**Reply by CharString** [reply]
Oh, the link in the show notes on Spotify is pointing here...

**Reply by codereport** [reply]
Ah thanks! Fixed.
